Documentation/methods/key.md

```diff
@@ -7,11 +7,15 @@ Returns information regarding given key.
 - key
 
 ## Example Response
-```php
+```js
 {
-    "key": "", 
-    "ownerUuid": "", // not dashed
-    "totalQueries": 142028083,
-    "queriesInPastMin": 68
-} 
-```
\ No newline at end of file
+  "success": true,
+  "record": {
+    "key": "e1513542-f4c7-483d-bf1b-3b29d4e59903",
```

Comment:
You leaked your API key here?
```suggestion
    "key": "",
```

Reply:
It's a random v4 uuid intentionally to indicate that it's dashed

Reply:
Ah, my bad haha. Makes sense to include this, though